source/logs.txt

```diff
@@ -69,5 +69,6 @@ automatically deletes log files older than 30 days.
 .. toctree::
    :titlesonly:
 
+   Toggle Logging </logs/disable>
```

Comment:
Is there a reason why we want this to defer from the docs page title which is "Disable logging"? I'm wondering if it is better to be consistent here.

Reply:
Ah thanks for catching this. I had originally called the page "toggle logging", but ended up changing it and forgot to update here. Will fix!